len_tests.py

```diff
@@ -0,0 +1,38 @@
+list = [1, 2, 3, 4, 5]
```

Comment:
FYI, it's generally a bad idea to name a variable the same thing as [a built-in function](https://docs.python.org/3/library/functions.html).

The function* [list](https://docs.python.org/3/library/functions.html#func-list) can be used to cast variables of other types to a list.

If you create a variable list, you make it so that function can no longer easily be called.

```python
>>> a_tuple = ('a', 'b', 'c')
>>> a_list = list(a_tuple)
>>> list = ['d', 'e', 'f']
>>> another_tuple = ('g', 'h', 'i')
>>> another_list = list(a_tuple)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'list' object is not callable
```

FWIW, I did almost exactly the same thing this week by creating my own function called round.

<small>* Technically list isn't a function, but that kind of pedantry can be pretty safely ignored 99.999…% of the time.</small>